.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 python:
   - 2.7
-  - 3.4
```

Comment:
I had issues getting the tests to run, because most of the test packages dropped python 3.4 support, while still supporting 2.7. pyyaml dropped support for python 3.4 with version 5.2 though that isn't documented in their change log. If python 3.4 support is required I could make the necessary changes.

Reply:
I think if it's OK to drop support for 3.4 at this point. I just wanted to make sure it wasn't due by something "fixable". Thanks for communicating your reasons for the drop.